README.md

```diff
@@ -1,193 +1,266 @@
+#### Deterministic ODE simulation of Michaelis-Menten enzyme kinetics
+Here we show a simple example where a model is created using the Catalyst DSL, and then simulated as 
+an ordinary differential equation.
 
 ```julia
-using Catalyst, Plots, JumpProcesses
-rs = @reaction_network begin
-  c1, S + E --> SE
-  c2, SE --> S + E
-  c3, SE --> P + E
+# Fetch required packages.
+using Catalyst, OrdinaryDiffEq, Plots
+
+# Create model.
+model = @reaction_network begin
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
 end
-p  = (:c1 => 0.00166, :c2 => 0.0001, :c3 => 0.1)
-tspan = (0., 100.)
-u0 = [:S => 301, :E => 100, :SE => 0, :P => 0]
-
-# solve JumpProblem
-dprob = DiscreteProblem(rs, u0, tspan, p)
-jprob = JumpProblem(rs, dprob, Direct())
-jsol = solve(jprob, SSAStepper())
-plot(jsol; lw = 2, title = "Gillespie: Michaelis-Menten Enzyme Kinetics")
+
+# Create an ODE that can be simulated.
+u0 = [:S => 50, :E => 10, :SE => 0, :P => 0]
+tspan = (0., 200.)
+ps = (:kB => 0.01, :kD => 0.1, :kP => 0.1)
+ode = ODEProblem(model, u0, tspan, ps)
+
+# Simulate ODE and plot results.
+sol = solve(ode)
+plot(sol; lw = 5)
 ```
+![ODE simulation](docs/src/assets/readme_ode_plot.svg)
 
-![](https://user-images.githubusercontent.com/1814174/87864114-3bf9dd00-c932-11ea-83a0-58f38aee8bfb.png)
+#### Stochastic jump simulations
+The same model can be used as input to other types of simulations. E.g. here we instead perform a 
+jump simulation
+```julia
+# Create and simulate a jump process (here using Gillespie's direct algorithm).
+using JumpProcesses
+dprob = DiscreteProblem(model, u0, tspan, ps)
+jprob = JumpProblem(model, dprob, Direct())
+jump_sol = solve(jprob, SSAStepper())
+plot(jump_sol; lw = 2)
+```
+![Jump simulation](docs/src/assets/readme_jump_plot.svg)
 
-#### Adaptive time stepping SDEs for a birth-death process
 
+## Elaborate example
+In the above example, we used basic Catalyst-based workflows to simulate a simple model. Here we 
+instead show how various Catalyst features can compose to create a much more advanced model. Our 
+model describes how the volume of a cell ($V$) is affected by a growth factor ($G$). The growth 
+factor only promotes growth while in its phosphorylated form ($Gᴾ$). The phosphorylation of $G$ 
+($G \to Gᴾ$) is promoted by sunlight (modeled as the cyclic sinusoid $kₐ*(sin(t)+1)$), which
+phosphorylates the growth factor (producing $Gᴾ$). When the cell reaches a critical volume ($V$)
+it undergoes through cell division. First, we declare our model:
 ```julia
-using Catalyst, Plots, StochasticDiffEq
-rs = @reaction_network begin
-  c1, X --> 2X
-  c2, X --> 0
-  c3, 0 --> X
+using Catalyst
+cell_model = @reaction_network begin
+    @parameters Vₘₐₓ g Ω
+    @default_noise_scaling Ω
+    @equations begin
+        D(V) ~ g*Gᴾ
+    end
+    @continuous_events begin
+        [V ~ Vₘₐₓ] => [V ~ V/2]
+    end
+    kₚ*(sin(t)+1)/V, G --> Gᴾ
+    kᵢ/V, Gᴾ --> G
```

Comment:
Some explanation of what is going on here / and or showing the SDEs would probably be useful for users.